src/Change.php

```diff
@@ -568,6 +568,51 @@ public function rawSearchOptions()
 
         $tab = array_merge($tab, ChangeCost::rawSearchOptionsToAdd());
 
+        $tab[] = [
+            'id'                 => 'ticket',
+            'name'               => Ticket::getTypeName(Session::getPluralNumber())
+        ];
+
+        $tab[] = [
+            'id'                 => '164',
+            'table'              => 'glpi_changes_tickets',
+            'field'              => 'id',
+            'name'               => _x('quantity', 'Number of tickets'),
+            'forcegroupby'       => true,
+            'usehaving'          => true,
+            'datatype'           => 'count',
+            'massiveaction'      => false,
+            'joinparams'         => [
+                'jointype'           => 'child'
+            ]
+        ];
+
+        return $tab;
+    }
+
+    public static function rawSearchOptionsToAdd()
+    {
+        $tab = [];
+
+        $tab[] = [
+            'id'                 => 'change',
+            'name'               => __('Changes')
+        ];
+
+        $tab[] = [
+            'id'                 => '210',
+            'table'              => 'glpi_changes_tickets',
+            'field'              => 'id',
+            'name'               => _x('quantity', 'Number of changes'),
+            'forcegroupby'       => true,
+            'usehaving'          => true,
+            'datatype'           => 'count',
+            'massiveaction'      => false,
+            'joinparams'         => [
+                'jointype'           => 'child'
+            ]
+        ];
+
```

Comment:
It should probably be added only for tickets (see how it is done in the `Problem` class).

```suggestion
    public static function rawSearchOptionsToAdd(string $itemtype)
    {
        $tab = [];

        if ($itemtype == "Ticket") {
            $tab[] = [
                'id'                 => 'change',
                'name'               => __('Changes')
            ];

            $tab[] = [
                'id'                 => '210',
                'table'              => 'glpi_changes_tickets',
                'field'              => 'id',
                'name'               => _x('quantity', 'Number of changes'),
                'forcegroupby'       => true,
                'usehaving'          => true,
                'datatype'           => 'count',
                'massiveaction'      => false,
                'joinparams'         => [
                    'jointype'           => 'child'
                ]
            ];
        }

```